pkg/genericapiserver/genericapiserver.go

```diff
@@ -892,6 +894,25 @@ func (s *GenericAPIServer) InstallSwaggerAPI() {
 		},
 	}
 	swagger.RegisterSwaggerService(swaggerConfig, s.HandlerContainer)
+	openAPIConfig := openapi.Config{
```

Comment:
This probably needs to be optional - I don't think we always want to include this, and this is not generic (in the way that apiserver is generic).  I think this needs to be in pkg/master.go, or have something that splits it out of this code path (where it can be invoked optionally.